core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4549,7 +4549,6 @@ components:
         properties:
           type: object
           description: A map of key-value pairs to associate with the snapshot. Optional, defaults to an empty map if not specified.
-          default: {}
```

Comment:
@aherbst-broad  i believe you added this [here](https://github.com/broadinstitute/rawls/commit/6f95120cc5cabc3cdeba1e2bba5c90d54acbaafc#diff-6e80a735b01ebcbdc253d755a6078cc1af5a585a6bd04b92b6455a9469988dffR4417). Does this always need to exist in the payload no matter what or can it be `null` by default?

Reply:
Should be fine, it's an option in the [model class](https://github.com/broadinstitute/rawls/blob/develop/model/src/main/scala/org/broadinstitute/dsde/rawls/model/DataReferenceModel.scala#L17)